.pre-commit-config.yaml

```diff
@@ -28,7 +28,7 @@ repos:
         types: [pyi]
         args: ["--profile", "black"]
   - repo: https://github.com/psf/black
-    rev: 21.12b0
+    rev: 22.3.0
```

Comment:
Looks like `.pre-commit-config.yaml` hasn't been kept in sync with requirements.txt as it should be.